### ISSUE TITLE
WebSocketGate does not validate webtoken

### ISSUE THREAD

**Issue by relu91** [arces-wot/SEPA#45]
**Describe the bug**
SEPA engine launched in secure mode does not validate subscription request.

**To Reproduce**
1. Start SEPA with the secure option set to true
2. Create a secure WebSocket connection with SEPA engine
3. Send a subscribe message
3. SEPA replies with the query results

**Expected behavior**
SEPA should reply with the error message specified [here](http://mml.arces.unibo.it/TR/sparql11-se-protocol.html#ErrorResponses)

**System information(please complete the following information):**
 - OS: Windows
 - Engine version: 0.9.7 and before
 - SparqlEnpoint: Blazegraph 2.14

**Comment by fr4ncidir** [comment]
I've recently re-written C apis and Python apis, I never encountered the problem since all requests were answered. Any more details?

**Comment by relu91** [comment]
Sure!
As stated in the issue description, consider that you have a secure WebSocket connection with SEPA. Consequently, send on this connection the following message:
```json
{"subscribe" : {
   "sparql" : "select * where {?vaimee ?deda ?didi}",
    "alias" : "ALL"
 }}
```
SEPA wrongly answers with a notification message with the results of the query. However, it should reply with an error message since the subscription request lacks the authorization field. This field is fundamental to have a valid aouth2 flow. 
The expected response message should be:
```json
{
  "error":"unauthorized_client",
  "error_description" : "Client is not authorized",
  "status_code" : 401
}
```

If you followed the specifications you probably didn't encountered in this issue, but still is there. 
Right now every client could basically read the entire database even though is not authorized by the system. 

I'll fix this issue next week. 

There were also problems with token expiration, before it wasn't validated. Be careful, the 60f49222cc4687d6d44839b396e87ac5b7467d05 change introduces token expiration validation. That means, if the token is expired it will be rejected and the client should renew it. This behavior was described in the various documents so your apis shouldn't be affected

**Comment by relu91** [comment]
Fixed f236823a959352ee6877f4d64ca603d2f95ec4b5
